src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1452,6 +1452,39 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode)
                                         ->variableNode('md5')->end()
                                     ->end()
                                 ->end()
+                                ->arrayNode('retry_strategy')
+                                    ->fixXmlConfig('http_code')
+                                    ->canBeEnabled()
+                                    ->addDefaultsIfNotSet()
+                                    ->beforeNormalization()
+                                        ->always(function ($v) {
+                                            if (isset($v['service']) && (isset($v['max_retries']) || isset($v['delay']) || isset($v['multiplier']) || isset($v['max_delay']))) {
+                                                throw new \InvalidArgumentException('The "service" cannot be used along with the other "retry_strategy" options.');
```

Comment:
this should be `The "service" option cannot...`, otherwise the sentence looks weird.

Reply:
indeed.. Copy/pasted from messenger configuration

Reply:
then maybe a bugifx PR is needed there too :)